api/config/feature-toggles-config.js

```diff
@@ -17,6 +17,12 @@ export default {
     defaultValue: true,
     tags: ['team-prescription', 'frontend'],
   },
+  enableAnonymisationWithDeletion: {
```

Comment:
```suggestion
  enableAnonymizationWithDeletion: {
```

Reply:
Je dirais même plus : 
```suggestion
  anonymizationWithDeletionEnabled: {
```   

Pour essayer d'être cohérent avec les autres toggles